test/distributed/org/apache/cassandra/distributed/impl/InstanceConfig.java

```diff
@@ -208,10 +216,9 @@ public void propagate(Object writeToConfig, Map<Class<?>, Function<Object, Objec
         throw new IllegalStateException("In-JVM dtests no longer support propagate");
     }
 
+    @Override
     public void validate()
     {
```

Comment:
Consider leaving a breadcrumb comment as to why we're leaving this here as a no-op after removing its body (precedent for this in our various ALTER/SELECT/etcStatement.java files in validate())